packages/babel-generator/src/generators/statements.ts

```diff
@@ -50,9 +50,20 @@ export function IfStatement(this: Printer, node: t.IfStatement) {
 }
 
 // Recursively get the last statement.
-function getLastStatement(statement) {
-  if (!isStatement(statement.body)) return statement;
-  return getLastStatement(statement.body);
+function getLastStatement(statement: t.Statement): t.Statement {
```

Comment:
```suggestion
function getLastStatement(statement: Exclude<t.Statement, t.BreakStatement>): t.Statement {
```

Does this help with the `// @ts-ignore`s below, or it just moves the problem to where we call this function?